misk/src/test/kotlin/misk/digester/VeneurDigestTest.kt

```diff
@@ -0,0 +1,108 @@
+package misk.digester
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+
+class VeneurDigestTest {
+
+  @Test
+  fun testVeneurDigest() {
+
+    var digest = VeneurDigest().newDefaultVeneurDigest()
+    assertThat(digest.mergingDigest.quantile( 0.5)).isEqualTo(Double.NaN)
+    assertThat(digest.sum()).isEqualTo(Double.NaN)
+    assertThat(digest.count()).isEqualTo(0)
+
+    digest.add(10.0)
+    digest.add(20.0)
+    assertThat(digest.mergingDigest.quantile(0.5)).isEqualTo(15.0)
+    assertThat(digest.count()).isEqualTo(2)
+    assertThat(digest.sum()).isEqualTo(30.0)
+
+  }
+
+  interface TestCasesStructure {
+    var sourceVals: Array<Double>
+    var destVals: Array<Double>
+    var expectedMedian: Double
+    var expectedSum: Double
+  }
+
+  @Test
+  fun testVeneurDigest_MergeInto() {
+    val testCases: Array<TestCasesStructure>
```

Comment:
I don't know Kotlin well enough to know the best conventions for table/parameterized tests. I suspect it's notably different since this will report as one unit test, not unit test per case in the `testCases` array.

That said, a few things I would change:
* Rename `TestCasesStructure` to make it less generic-sounding.
* Make `TestCasesStructure` private.
* Make `TestCasesStructure` a data class (https://kotlinlang.org/docs/reference/data-classes.html) instead of an interface.